packages/server/graphql/mutations/startTeamPrompt.ts

```diff
@@ -0,0 +1,79 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql'
+import getRethink from '../../database/rethinkDriver'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import StartTeamPromptPayload from '../types/StartTeamPromptPayload'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {GQLContext} from '../graphql'
+import standardError from '../../utils/standardError'
+import isStartMeetingLocked from './helpers/isStartMeetingLocked'
+import {MeetingTypeEnum} from '../../postgres/types/Meeting'
+import createNewMeetingPhases from './helpers/createNewMeetingPhases'
+import generateUID from '../../generateUID'
+import MeetingTeamPrompt from '../../database/types/MeetingTeamPrompt'
+import {startMattermostMeeting} from './helpers/notifications/notifyMattermost'
+import {startSlackMeeting} from './helpers/notifications/notifySlack'
+import sendMeetingStartToSegment from './helpers/sendMeetingStartToSegment'
+
+const startTeamPrompt = {
+  type: GraphQLNonNull(StartTeamPromptPayload),
+  description: ``,
```

Comment:
\-1 description was missing

Reply:
it's there: https://github.com/ParabolInc/parabol/pull/6171/files#diff-49ad6d7c7278b17294131a1bceddfb74714031fee6e4433287256dd997708d50R24